src/en/juju-gui-management.md

```diff
@@ -2,87 +2,76 @@ Title: The Juju GUI
 
 # The Juju GUI
 
-Juju has a graphical user interface (GUI) available to help with the tasks of
-managing and monitoring your Juju environment. The GUI is a JavaScript and HTML
-web application that is encapsulated in its own charm, which can be installed
-to its own service or alongside another service. Once installed, the GUI will
-talk with Juju over a websocket to provide a real-time interface with the
-services installed, the units that comprise them, and the machines available.
-Additionally, the GUI can talk with the charm store in order to search, browse,
+Juju has a graphical user interface (GUI) available to help with the tasks of 
+managing and monitoring your Juju environment. From Juju 2.0 onwards, unless 
+you bootstrap your controller with the `--no-gui` option, the GUI is 
+automatically embedded within every Juju controller.
+
+The GUI is a JavaScript and HTML web application that is encapsulated in its 
+own charm. Once installed, the GUI will talk with Juju over a websocket to 
+provide a real-time interface with the services installed, the units that 
+comprise them, and the machines available. Additionally, the GUI can talk with 
+the charm store in order to search, browse,
 and deploy charms to your environment.
 
+# Using the GUI
 
-# Installation
+Juju GUI is now automatically included in every Juju controller after 
+bootstrapping, thus eliminating the need to additionally deploy the Juju GUI 
+charm. As it's part of he controller, it won't appear as a normally deployed 
+service.
 
-Installing the GUI works the same as installing any other charm:
+To open Juju GUI in your default browser, use following command:
 
-```bash
-juju deploy juju-gui
-juju expose juju-gui
 ```
```

Comment:
code fencing needs to define a type - it is a sad limitation of our syntax highlighting.

Usually you will use \```bash